test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/ExitOnProcessExitTests.cs

```diff
@@ -83,7 +84,7 @@ public sealed class TestAssetFixture() : TestAssetFixtureBase(AcceptanceFixture.
 using Microsoft.Testing.Platform.Extensions.Messages;
 using Microsoft.Testing.Platform.Extensions.TestFramework;
 
-if (args.Length == 0)
+if (!args.Contains("--exit-on-process-exit"))
```

Comment:
This test assumed no args, but we now set --no-progress --no-ansi, so it never got setup, and was just sitting there stuck.